cmd/start.go

```diff
@@ -144,5 +149,53 @@ func newStartCommand(ctx context.Context, args []string) *cobra.Command {
 }
 
 
+func encryptConfig(encryptPass string) error{
+	var s = make([]string,2)
+	s[0] = "password:"
+	s[1] = "encrypted:" + encryptPass
+	confPath := filepath.Join(viper.GetString(flagFileServerPath),defaultConfigFileName)
```

Comment:
this will not necessarily be the config file path as this is only the default and can be overridden. Maybe it is better to use viper.ConfigFileUsed() or the configFilePath variable in NewStartCommand?

Reply:
Done